aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -104,15 +104,19 @@ def query_string_parameters(self) -> Optional[Dict[str, str]]:
         return self.get("queryStringParameters")
 
     @property
-    def resolved_query_string_parameters(self) -> Optional[Dict[str, str]]:
+    def resolved_query_string_parameters(self) -> Optional[Dict[str, List[str]]]:
         """
         This property determines the appropriate query string parameter to be used
         as a trusted source for validating OpenAPI.
 
         This is necessary because different resolvers use different formats to encode
         multi query string parameters.
         """
-        return self.query_string_parameters
+        if self.query_string_parameters is not None:
```

Comment:
maybe we keep in common to avoid forgetting a resolver when upgrading/improving the resolution?